administrator/components/com_content/src/Model/ArticlesModel.php

```diff
@@ -147,6 +172,8 @@ protected function populateState($ordering = 'a.id', $direction = 'desc')
         $this->getUserStateFromRequest($this->context . '.filter.access', 'filter_access');
         $this->getUserStateFromRequest($this->context . '.filter.language', 'filter_language', '');
         $this->getUserStateFromRequest($this->context . '.filter.checked_out', 'filter_checked_out', '');
+        $this->getUserStateFromRequest($this->context . '.filter.tag_mode', 'filter_tag_mode', 'any');
+
```

Comment:
```suggestion
        $this->getUserStateFromRequest($this->context . '.filter.tag_mode', 'filter_tag_mode', 'any');
```
Remove redundant empty line.

Reply:
done